src/App.css

```diff
@@ -0,0 +1,68 @@
+* {
+  margin: 0;
+ }
+ .app {
+  background-color: #fafafa;
+  height: 130vh;
+  
+}
+.component6 {
+  display: flex;
+  flex-direction: column;
+  position:absolute;
+  border: 1px solid lightgray;
+  top:100%;
+  right:50px;
+  transform:translateY(-130%);
+  max-width: 500px;
+  padding: 10px;
+}
+
+.title {
+  margin-left: auto;
+  margin-right: auto;
+  padding: 10px;
+}
+.date {
+  margin-left: auto;
+  margin-right: auto;
+  padding: 10px;
+}
+
+.players {
+  object-fit: contain;
+  padding: 20px;
+}
```

Comment:
It looks like all this css already exists in the `TopPlayerFeed` component's css file. Is the intention to delete it from that file and move it into this one? It doesn't look like any of these classes are used in the `Header` or `App` component. Even with external stylesheets I suggest we aim to keep the css modular and relevant specifically to the component it's for (which might also involve moving the header css into a separate css file).

On a separate note, I might also suggest we come up with a naming convention as a group for our styling. I think [BEM](http://getbem.com/naming/) tends to be pretty popular, or we can switch to using something like [Tailwind](com) that prevents us from having to write css ourselves.

Reply:
Thanks, I didn't notice that the css file got changed, so that's adjusted.  I'm all for going with a naming convention going forward, I'm just not familiar with using any of them.